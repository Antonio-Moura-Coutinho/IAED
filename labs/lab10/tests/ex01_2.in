lknw duj furvq aqxi hleiw jih hkgmc crbm zygmd mgfyk rjed vwz wel rlouw dmi gqsw kseh kwt yhecw jwsf ywzq xtwl hynu lstuc jcovm ofaz vzlk rctme zgbk tpfov twyb afps zkft alo fetwa rvg ejpr psn jlym zwk dcy qvprj hecxj pzh ofv vug qdbug mqpc gzkxp atn omd lyrk yfejt tase cqwuf lxea yfb uvde kas bvlz cyl idl jtn uelyb efxai kwnl dxjv kbat ude ysnlr maplh puban acwds sazg qim oku ivsem nxfij iqzs pbzx hfq nmq xstb xzu lfh izpyk qtc kmtvc ylf jvkm kdnxw vgkci mwroz xpaw utp xks gjpit kol lmof teyza x