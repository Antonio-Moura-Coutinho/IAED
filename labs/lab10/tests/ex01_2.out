teyza
lmof
kol
gjpit
xks
utp
xpaw
mwroz
vgkci
kdnxw
jvkm
ylf
kmtvc
qtc
izpyk
lfh
xzu
xstb
nmq
hfq
pbzx
iqzs
nxfij
ivsem
oku
qim
sazg
acwds
puban
maplh
ysnlr
ude
kbat
dxjv
kwnl
efxai
uelyb
jtn
idl
cyl
bvlz
kas
uvde
yfb
lxea
cqwuf
tase
yfejt
lyrk
omd
atn
gzkxp
mqpc
qdbug
vug
ofv
pzh
hecxj
qvprj
dcy
zwk
jlym
psn
ejpr
rvg
fetwa
alo
zkft
afps
twyb
tpfov
zgbk
rctme
vzlk
ofaz
jcovm
lstuc
hynu
xtwl
ywzq
jwsf
yhecw
kwt
kseh
gqsw
dmi
rlouw
wel
vwz
rjed
mgfyk
zygmd
crbm
hkgmc
jih
hleiw
aqxi
furvq
duj
lknw
